docs/content/docs/dev/table/sql-gateway/overview.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Overview
+weight: 1
+type: docs
+aliases:
+- /dev/table/sql-gateway.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+Introduction
+----------------
+
+The SQL Gateway is a service that enables multiple clients from the remote to execute SQL in concurrency. It provides 
+an easy way to submit the Flink Job, look up the metadata, and analyze the data online.
+
+The SQL Gateway is composed of pluggable endpoints and the `SqlGatewayService`. The `SqlGatewayService` is a processor that is 
+reused by the endpoints to handle the requests. The endpoint is an entry point that allows users to connect. Depending on the 
+type of the endpoints, users can use different utils to connect.
+
+{{< img width="80%" src="/fig/sql-gateway-architecture.png" alt="SQL Gateway Architecture" >}}
```

Comment:
Update the picture with more tools, such as Hive Beeline CLI, DolphinScheduler, Zepplin.